in_toto/model.go

```diff
@@ -150,6 +190,67 @@ func (l *Layout) InspectAsInterfaceSlice() []interface{} {
 	return inspectionsI
 }
 
+func (l *Layout) validate() {
+	validateFunctions := []func(){l.validateType, l.validateExpires,
+		l.validateKeys, l.validateStepsAndInspections}
+	for _, f := range validateFunctions {
+		f()
+	}
+}
+
+func (l *Layout) validateType() {
+	if l.Type != "layout" {
+		fmt.Println("Invalid Type value for layout: should be 'layout'")
+	}
+}
+
+func (l *Layout) validateExpires() {
+	if _, err := time.Parse("2006-01-02T15:04:05Z", l.Expires); err != nil {
```

Comment:
We should have a constant for our own *profile* of ISO 8601, which is similar to RFC 3339 but without trailing offset (as you correctly hard-coded here). In `securesystemslib` it is called [`ISO8601_DATETIME_SCHEMA`](https://github.com/secure-systems-lab/securesystemslib/blob/v0.11.2/securesystemslib/formats.py#L93). The same one could then be used in `verifylib.VerifyLayoutExpiration`.

Reply:
Okay, I'll do that. It'll just be set to the same value I've used, as that's what `time.Parse` expects.